web_view_rtree/readme/ROADMAP.rst

```diff
@@ -0,0 +1,20 @@
+This module is in an alpha state. It is still being developed and should be
+considered as experimental.
```

Comment:
```suggestion
```

``    "development_status": "Alpha",`` is enough. It will add an explicit warning in the README.rst file. It will be removed also automatically, when the status will change.

Reply:
thanks for your suggestions! done.